docs/modules/reference/pages/service-assurance/monitors/TrivialTimeMonitor.adoc

```diff
@@ -1,8 +1,10 @@
 
 = TrivialTimeMonitor
+:description: Learn about the TrivialTimeMonitor in OpenNMS Horizon/Meridian to send the time as a 32-bit unsigned integer in binary format, in network byte order.
+// why do they want to do this? What is the purpose?
```

Comment:
I don't know a specific use case for this, but there are some systems that use Time Protocol instead of NTP.